### ISSUE TITLE
Add a "dark mode" to go with the VS code theme

### ISSUE THREAD

**Issue by tgross35** [42Crunch/vscode-openapi#144]
Hey all,

It would be awesome to have a dark mode for the swaggerui renderer. This isn't native in swaggerui, but it may be possible to implement in the OpenAPI extension.

Someone details how to do a dark mode here: https://dev.to/amoenus/turn-swagger-theme-to-the-dark-mode-4l5f

And there is a usertheme here: https://uso.kkx.one/style/188706

Love the extension, would be awesome to see in dark mode!

**Comment by ak1394** [comment]
Thanks for the feedback. I'll check to see what we can do about it, we've recently upgraded to a most recent version of SwaggerUI and I'm not sure if these examples play nice with it. Swagger UI doesn't yet support dark mode directly https://github.com/swagger-api/swagger-ui/issues/5327

**Comment by Robouste** [comment]
Is there any update about implementing a dark mode?

**Comment by Droutin** [comment]
up

**Comment by Buzut** [comment]
Yes do please, working at night I can't even use the preview tab, it's violent on the eyes!

**Comment by Droutin** [comment]
up
